public/templates/demo.html

```diff
@@ -18,7 +18,7 @@
     <link
       rel="shortcut icon"
       type="image/x-icon"
-      href="https://cdn.glitch.com/aa3eaa29-7183-4f33-b4e2-c3a30c833a25%2F31540120.jpg?v=1598133005255"
+      href="/public/assets/images/favicon.jpg"
```

Comment:
This should be /relative/ to public:

`/assets/images/favicon.jpg`

Fix this every other place as well. The website root is `/public` and the `assets` folder is under that.